#Goal
string group
moveit_msgs/RobotTrajectory plan
bool force_active
float32 force_limit
---------------
#Result
bool success
---------------
#Feedback
bool done
int8 step_inc